docs/guides/azure-private-link-workspace-standard.md

```diff
@@ -0,0 +1,512 @@
+---
+page_title: "Provisioning Databricks on Azure with Private Link - Standard deployment"
+---
+
+# Deploying pre-requisite resources and enabling Private Link connections - Standard deployment
+
+-> **Note** 
+  - Refer to [adb-with-private-link-standard](https://github.com/databricks/terraform-databricks-examples/tree/main/modules/adb-with-private-link-standard), a Terraform module that contains code used to deploy an Azure Databricks workspace with Azure Private Link using the Standard deployment approach. 
+  - Refer to the [Databricks Terraform Registry modules](https://registry.terraform.io/modules/databricks/examples/databricks/latest) for more Terraform modules and examples to deploy Azure Databricks resources.
+  - This guide assumes that connectivity from the on-premises user environment is already configured using ExpressRoute or a VPN gateway connection.
+
+Databricks Private Link support enables private connectivity between users and their Databricks workspaces and between clusters on the data plane and core services on the control plane within the Databricks workspace infrastructure. 
+
+You can use Terraform to deploy the underlying cloud resources and the private access settings resources automatically, using a programmatic approach. 
+
+
+This guide covers a [standard deployment](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/private-link-standard) to configure Azure Databricks with Private Link:
+* Two seperate VNets are used:
+  * A transit VNet 
+  * A customer Data Plane VNet
+* A private endpoint is used for back-end connectivity and deployed in the customer Data Plane VNet.
+* A private endpoint is used for front-end connectivity and deployed in the transit VNet.
+* A private endpoint is used for web authentication and deployed in the transit VNet.
+* A dedicated Databricks workspace, called Web Auth workspace, is used for web authentication traffic. This workspace is configured with the sub resource **browser_authentication** and deployed using subnets in the transit VNet.
+
+-> **Note**  
+* A seperate Web Auth workspace is not mandatory but recommended.
+* DNS mapping for SSO login callbacks to the Azure Databricks web application can either be managed by the Web Auth workspace or another workspace that is associated with the **browser_authentication** private endpoint.
+
+![Azure Databricks with Private Link - Standard deployment](https://github.com/databricks/terraform-provider-databricks/raw/master/docs/images/azure-private-link-standard.png)
+
+This guide uses the following variables:
+
+- `cidr_transit`: The network range (CIDR) for the Azure transit VNet
```

Comment:
what are the minimum VNet size for these?

Reply:
Normally a CIDR block between /16 and /24 for the VNets. 
But we deploy private endpoints and we deploy a workspace in both vnets. One of them is for browser_auth. 
I think /23 should be the minimum.